tensorflow/core/kernels/depthwise_conv_op_gpu.cu.cc

```diff
@@ -1015,6 +1016,21 @@ __global__ void __launch_bounds__(640, 2)
   }
 }
 
+// Device function to compute sub-warp reductions for a power-of-two group of
```

Comment:
"reduction" is a generic term -- what is being done here is a *sum* reduction.  I think we should change the comment, but also the name of the function.